src/serializers/encoder/v16/encode_graph_entities.c

```
@@ -0,0 +1,505 @@
+	SerializerIO rdb,
+	const SIValue *v
+) {
+	// Format:
+	// SIType
+	// Value
+
+	SerializerIO_WriteUnsigned(rdb, v->type);
+
+	switch(v->type) {
+		case T_BOOL:
+		case T_INT64:
```

cppcheck

[error] Memory leak

(memleak)